.github/workflows/test.yml

```diff
@@ -50,11 +50,18 @@ jobs:
             include: "sale_procurement_group_by_commitment_date"
             name: test with OCB
           - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
-            exclude: "sale_order_line_sequence,sale_procurement_group_by_commitment_date"
+            include: "sale_global_discount"
             makepot: "true"
             name: test with Odoo
           - container: ghcr.io/oca/oca-ci/py3.6-ocb14.0:latest
-            exclude: "sale_order_line_sequence,sale_procurement_group_by_commitment_date"
+            include: "sale_global_discount"
```

Comment:
do we really need this exception?

Reply:
@bguillot Could you answer this ?

Reply:
it seems that there is a bug in sale_global_discount when having several inclueded taxes : https://github.com/OCA/sale-workflow/issues/2123